### DISCUSSION TITLE
Access for workstations without /trust

### DISCUSSION THREAD

**Discussion by Sanszi** [GriefPrevention/GriefPrevention#1770] [Ideas]
On our server we really miss a command that would let non trusted players use workstations such as: anvil, grindstone, smithing table etc. 

I'd be grateful if you could implement it.

**Comment by Jikoo** [comment]
Anvils specifically require higher trust because they break over time. Would be nice to have a config node for that I suppose. Everything else should be accessible via `/containertrust public`, which can be done on a small subclaim over the public access area. An addon can also change this behavior via the `ClaimPermissionCheckEvent`.

**Reply by Sanszi** [reply]
Thanks for the response. You are right, everything works with the /containertrust command, but the anvil. I hope we'll find a solution to it eventually. Thank you again!
